cli/.pylintrc

```diff
@@ -158,6 +158,8 @@ disable=print-statement,
         no-self-use,
         no-member,
         unused-import, # handled by flake8
+        no-else-return, # A complete if else block can improve readability even with returns in the body
+        invalid-name, # This would prevent us from using single letter variable
```

Comment:
I'd keep this check https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions

Reply:
Done